utils/src/main/java/com/cloud/utils/net/NetUtils.java

```diff
@@ -775,11 +776,19 @@ public static String getSubNet(final String ip, final String netmask) {
     }
 
     public static String getCidrSubNet(final String ip, final long cidrSize) {
-        final long numericNetmask = 0xffffffff >> MAX_CIDR - cidrSize << MAX_CIDR - cidrSize;
+        final long numericNetmask = netMaskFromCidr(cidrSize);
         final String netmask = NetUtils.long2Ip(numericNetmask);
         return getSubNet(ip, netmask);
     }
 
+    /**
+     * @param cidrSize
+     * @return
+     */
+    static long netMaskFromCidr(final long cidrSize) {
+        return ((long)0xffffffff) >> MAX_CIDR - cidrSize << MAX_CIDR - cidrSize;
+    }
+
```

Comment:
Why are tho static methods using "package" modifier? Are classes in the same package going to access it? If the answer is no, they can be made "private".